Makefile

```diff
@@ -1,23 +1,30 @@
 GIT_COMMIT ?= $(shell git rev-list -1 HEAD)
 GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
+LOCAL_DOCKER_BASE := local/dev_promscale_extension
+GHCR_DOCKER_BASE := ghcr.io/timescale/dev_promscale_extension
 
 .PHONY: build
 build: generate
 	go build -ldflags "-X 'github.com/timescale/promscale/pkg/version.Branch=${GIT_BRANCH}' -X 'github.com/timescale/promscale/pkg/version.CommitHash=${GIT_COMMIT}'" -o dist/promscale ./cmd/promscale
 
 .PHONY: test
-test: generate
-	go test -v -race ./... -timeout 40m
+test: unit e2e
+
+.PHONY: unit
+unit: generate
+	go test -v -race $(shell go list ./... | grep -v tests/end_to_end_tests) -timeout 40m
```

Comment:
I think you'll need to exclude upgrade_tests eventually as well

Reply:
Possibly. As I mentioned below, I think that we need to rework the upgrade tests a little.